test/state/mpt_hash.hpp

```diff
@@ -9,7 +9,16 @@
 namespace evmone::state
 {
 struct Account;
+struct Transaction;
+struct TransactionReceipt;
 
 /// Computes Merkle Patricia Trie root hash for the given collection of state accounts.
 hash256 mpt_hash(const std::unordered_map<address, Account>& accounts);
```

Comment:
Given https://github.com/ethereum/evmone/pull/569/files#r1110980443, maybe then we should rename this to `mpt_hash_accounts`.